stats/opentelemetry/e2e_test.go

```diff
@@ -127,12 +95,11 @@ func (s) TestMethodTargetAttributeFilter(t *testing.T) {
 		// Will allow duplex/any other type of RPC.
 		return str != "/grpc.testing.TestService/UnaryCall"
 	}
-	// pull out setup into a helper
-	reader, ss := setup(t, true, maf)
+	reader, ss := setup(t, maf)
 	defer ss.Stop()
 
-	// make a single RPC (unary rpc), and filter out the target and method
-	// that would correspond.
+	// make a single RPC (unary rpc), and filter out the method that would
```

Comment:
One unary and one streaming RPC is being made here.

Reply:
Added comment: "Make a Unary and Streaming RPC. The Unary RPC should be filtered by the method attribute filter, and the Full Duplex (Streaming) RPC should not."